### PR TITLE
push runner: correct the comment string in customization yaml files

### PR DESCRIPTION
**Pull request by jan--f**
PR cause I'd like someone to look at the indentation of the long line split here https://github.com/SUSE/DeepSea/compare/master...jan--f:master#diff-bc35b5fa836936a3136544887240f188R141

I also took the chance and cleand up some trailing whitespaces.

Signed-off-by: Jan Fajerski <jfajerski@suse.com>

### CODE REVIEW

File: srv/modules/runners/push.py [hunk 1]
```diff
@@ -137,9 +137,14 @@ def _custom(self, custom):
...
                 with open(custom, "w") as yml:
-                    yml.write("# Customizations for {}\n".format(custom))
-    
-    
+                    custom_split = custom.split("stack")
+                    custom_for = "{}{}{}".format(
+                            custom_split[0],
+                            "stack/default",
+                            custom_split[1])
+                    yml.write("# Customizations for {}\n".format(custom_for))
```
Comment (theanalyst) [inline srv/modules/runners/push.py]: We could probably do it in one shot as something like 
```py
yml.write("customizations for {0}{1}{2}\n".format(split[0]...)
```
Though that doesn't describe it any better :)

Comment (jan--f) [inline srv/modules/runners/push.py]: yeah, I had that initially but then the line gets even longer.

Comment (jan--f) [inline srv/modules/runners/push.py]: Where is pathname from?
In any case imho right now it sounds like this file overwrites the content in the default tree. Maybe we should add two comments? Something like:
`# This file will overwrite configuration made in /srv/pillar/ceph/stack/default/global.yml when put in /srv/pillar/ceph/stack/global.yml`

Comment (swiftgist) [inline srv/modules/runners/push.py]: line 96.  It's the keys of the relative filenames that made it this far.

maybe
# /srv/pillar/ceph/stack/global.yml
# Overwrites configuration in /srv/pillar/ceph/stack/default/global.yml

Comment (jan--f) [inline srv/modules/runners/push.py]: sure lgtm...will push a fix

**Pull request merged**